python/cudf_polars/cudf_polars/dsl/expr.py

```diff
@@ -826,6 +829,36 @@ def do_evaluate(
                     plc.interop.from_arrow(pa.scalar(stop, type=pa.int32())),
                 )
             )
+        elif self.name in {
+            pl_expr.StringFunction.StripChars,
+            pl_expr.StringFunction.StripCharsStart,
+            pl_expr.StringFunction.StripCharsEnd,
+        }:
+            column = self.children[0].evaluate(df, context=context, mapping=mapping)
+            expr_chars = self.children[1]
+            assert isinstance(expr_chars, Literal)
+
+            if expr_chars.value == pa.scalar(""):
+                # No-op in python, polars, but libcudf removes whitespace so return early
+                return column
```

Comment:
Could we observe this during `translation` and just return the child column expression?